css/styles.css

```diff
@@ -160,14 +160,15 @@ Layout styles
 	position: absolute;
 	left: 0;
 	margin-left: 1em;
+	max-width: 275px;
 }
 
 .right_bar
 {
 	position: absolute;
 	right: 0;
 	margin-right: 1em;
-	max-width: 25%;
+	max-width: 275px;
```

Comment:
I'd like to use a fixed pixel-value max width here, since on very wide screens `25%` can be unpleasantly wide to read text and obscure a meaningful part of the graph vis 😄 

for reference, I work on a 38" monitor and experience this % width problem 😅